packages/core-data/src/entity-provider.js

```diff
@@ -221,6 +221,15 @@ export function useEntityBlockEditor( kind, name, { id: _id } = {} ) {
 			);
 
 			function updateAttributes( attributes ) {
+				// Only attempt to update attributes, if attributes is an object.
+				if (
+					! attributes ||
```

Comment:
Note: this falsy check is required because `typeof null` equals `object` so this catches if somehow this function is called with `null`.

Reply:
I haven't tested this yet, but just staring at it makes me think it's a safe change regardless since we're destructuring `attributes` below.

Reply:
Yeah, one of the things that tripped me up when looking at this function is that when passed anything other than an object, the destructuring winds up converting to an object.

Hopefully it's a fairly safe-ish change! The thing that took me the longest was testing it out in a custom block 😅

Thanks for taking a look!